src/components/editions/headerImage.tsx

```diff
@@ -1,38 +1,100 @@
 // ----- Imports ----- //
 
+import type { SerializedStyles } from '@emotion/core';
 import { css } from '@emotion/core';
+import type { Sizes } from '@guardian/image-rendering';
+import { Img } from '@guardian/image-rendering';
 import { remSpace } from '@guardian/src-foundations';
-import Img from 'components/img';
+import { from } from '@guardian/src-foundations/mq';
+import { none, some } from '@guardian/types';
+import HeaderImageCaption, { captionId } from 'components/headerImageCaption';
 import { MainMediaKind } from 'headerMedia';
+import type { Image } from 'image';
 import type { Item } from 'item';
 import { maybeRender } from 'lib';
 import type { FC } from 'react';
+import { tabletContentWidth, wideContentWidth } from './styles';
 
 // ----- Component ----- //
 
 const styles = css`
-	margin: 0;
+	margin: 0 0 ${remSpace[2]} 0;
+	position: relative;
+
+	${from.tablet} {
+		width: ${tabletContentWidth}px;
+		margin-left: auto;
+		margin-right: auto;
+	}
+
+	${from.wide} {
+		width: ${wideContentWidth}px;
+		margin-left: auto;
+		margin-right: auto;
+	}
 `;
 
-const imgStyles = css`
-	display: block;
-	width: 100%;
+const captionStyles = css`
+	${from.tablet} {
+		width: ${tabletContentWidth}px;
+	}
+
+	${from.wide} {
+		width: ${wideContentWidth}px;
+	}
 `;
 
+const getImageStyle = ({ width, height }: Image): SerializedStyles => {
+	return css`
+		display: block;
+		width: 100%;
+		height: calc(100vw * ${height / width});
+
+		${from.tablet} {
+			width: ${tabletContentWidth}px;
+			height: ${(tabletContentWidth * height) / width}px;
+		}
+
+		${from.wide} {
+			width: ${wideContentWidth}px;
+			height: ${(wideContentWidth * height) / width}px;
+		}
+	`;
+};
+
 interface Props {
 	item: Item;
 }
 
+const sizes: Sizes = {
+	mediaQueries: [
+		{ breakpoint: 'tablet', size: '740px' },
+		{ breakpoint: 'wide', size: '980px' },
+	],
+	default: '100vw',
+};
```

Comment:
Is this worth moving sizes to editions' `style.ts` ? Seems like something that various components will share.

Reply:
If other components include images that match the header image size across all breakpoints then yes.